generators/app/templates/ci/.circleci/config.yml

```diff
@@ -60,21 +60,21 @@ jobs:
       - store_artifacts:
           path: htmlcov
 
-  black:
+  ruff-format:
     executor: base-python
     steps:
       - get_code_and_cache
       - run:
-          name: run black
-          command: make black
+          name: run ruff-format
```

Comment:
run ruff formatter

Reply:
This should actually do formatting right? I think before, it just did `black . --check`, so wouldn't have actually made formatting changes

Reply:
Sorry I was not clear, I wanted to tell to write "run ruff formatter", instead of "run ruff-format". Considering our discussion about naming in Makefile, I think we could finally write here "run formatter"